manifest_staging/charts/secrets-store-csi-driver/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: secrets-store-csi-driver-psp
+spec:
```

Comment:
We should also include the labels here similar to https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml#L7

Reply:
Done